doc/reference/reference_sql/sql_statements_and_clauses.rst

```diff
@@ -620,140 +620,95 @@ causes no error message, although (s1, s1) is probably a user error.
-    INSERT INTO t1 VALUES (2, 1);
-
-The optional update-or-delete rules look like this: |br|
-``ON {UPDATE|DELETE} { CASCADE | SET DEFAULT | SET NULL | RESTRICT | NO ACTION}`` |br|
-and the idea is: if something changes the referenced key, then one of these possible "referential actions" takes place: |br|
-``CASCADE``: the change that is applied for the referenced key is applied for the referencing key. |br|
-``SET DEFAULT``: the referencing key is set to its default value. |br|
-``SET NULL``: the referencing key is set to NULL. |br|
-``RESTRICT``: the UPDATE or DELETE fails if a referencing key exists; checked immediately. |br|
-``NO ACTION``: the UPDATE or DELETE fails if a referencing key exists; checked at statement end. |br|
-The default is ``NO ACTION``.
-
-For example:
+    {referencing_column_name} {column_definition}
+        REFERENCES {referenced_table_name}({referenced_column_name})
 
-.. code-block:: sql
+.. NOTE::
 
-    CREATE TABLE f1 (ordinal INTEGER PRIMARY KEY,
-                 referenced_planet STRING UNIQUE NOT NULL);
-    CREATE TABLE f2 (
-        ordinal INTEGER PRIMARY KEY,
-        referring_planet STRING DEFAULT 'Earth',
-        FOREIGN KEY (referring_planet) REFERENCES f1 (referenced_planet)
-            ON UPDATE SET DEFAULT
-            ON DELETE CASCADE);
-    INSERT INTO f1 VALUES (1, 'Mercury'), (2,' Venus'), (3, 'Earth');
-    INSERT INTO f2 VALUES (1, 'Mercury'), (2, 'Mercury');
-    UPDATE f1 SET referenced_planet = 'Mars'
-        WHERE referenced_planet = 'Mercury';
-    SELECT * FROM f2;
-    DELETE FROM f1 WHERE referenced_planet = 'Earth';
-    SELECT * FROM f2;
-    ... In this example, the UPDATE statement changes the referenced key,
-        and the clause is ON UPDATE SET DEFAULT, therefore both of the
-        rows in f2 have referring_planet set to their default value,
-        which is 'Earth'. The DELETE statement deletes the row that
-        has 'Earth', and the clause is ON DELETE CASCADE,
-        therefore both of the rows in f2 are deleted.
+    With :doc:`2.11.0 </release/2.11.0>`, the following referencing options aren't supported anymore:
 
-Limitations: |br|
-* Foreign keys can have a MATCH clause (`Issue#3455 <https://github.com/tarantool/tarantool/issues/3455>`_).
+    *   The ``ON UPDATE`` and ``ON DELETE`` triggers. The ``RESTRICT`` trigger action is used implicitly.
+    *   The ``MATCH`` subclause. ``MATCH FULL`` is used implicitly.
+    *   ``DEFERRABLE`` constraints. The ``INITIALLY IMMEDIATE`` constraint check time rule is used implicitly.
 
-.. COMMENT
-   Constraint Conflict Clauses are temporarily disabled.
-   However, the description is here, as a big comment.
 
-   Constraint Conflict Clauses
+The referenced column names must be defined in the referenced table and have similar data types.
+The values in the referencing columns must equal values in the referenced columns of the referenced table,
+or at least one of the referencing columns must contain ``NULL``.
```

Comment:
This is currently not the case. This describes `MATCH SIMPLE`, which is usually the default. However, we only support `MATCH FULL`, which requests a full match.